### PR TITLE
➕ Add stylelint dependency

### PR DESCRIPTION
**Pull request by fre-ben**
- Add stylelint
- Add stylelint script to `.package.json`
- Add stylelint/prettier config

https://stylelint.io/user-guide/get-started
https://github.com/prettier/stylelint-config-prettier

### CODE REVIEW

**Review by lmachens — APPROVED** [review]
🚀

**Review by phlgr — APPROVED** [review]
🚀

**Comment by DNaunin** [comment]
🚀

File: package.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
     "build": "next build",
     "start": "next start",
     "lint": "eslint .",
+    "stylelint": "stylelint '**/*.css'",
```
Comment (m-wiemers) [inline package.json:10]: You can`t run i think...

```suggestion
    "stylelint": "stylelint \"**/*.css\"",
```

this can help

**Review by m-wiemers — CHANGES_REQUESTED** [review]
gj 🚀

File: package.json (hunk 1 above)
Comment (fre-ben) [reply package.json:10]: Thanks for the suggestion, but it's working with the `'`'s :P

**Review by christophvogel — APPROVED** [review]
Perfect as always! 🥇

**Pull request merged**